OMEdit/OMEditLIB/MainWindow.cpp

```diff
@@ -75,7 +75,6 @@
 #include "Plotting/DiagramWindow.h"
 #include "Interfaces/InformationInterface.h"
 #include "Interfaces/ModelInterface.h"
-#include "omc_config.h"
 #include "Util/NetworkAccessManager.h"
 #include "Modeling/InstallLibraryDialog.h"
 #include "CrashReport/CrashReportDialog.h"
```

Comment:
This one seems to be needed. The definition of HOST_SHORT macro is in `omedit_config.h`

Reply:
Yes. The windows and linux config files are not the same :)

Reply:
We can fix that soon. Hopefully.